src/tests/userLogin.test.js

```diff
@@ -49,6 +49,38 @@ const usersignIn = () => {
         });
     });
   });
+  describe('User Logout', () => {
+    it('it should return 200 on successful logout', (done) => {
+      chai
+        .request(app)
+        .patch('/api/v1/auth/logout')
+        .send()
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(200);
```

Comment:
@Niyonsengaeric to make your tests more trustable, I suggest you also include the expected message if any, along the expected status. (apply to the rest of the tests)